pyrdp/mitm/BasePathMITM.py

```diff
@@ -60,4 +60,4 @@ def onScanCode(self, scanCode: int, isReleased: bool, isExtended: bool):
         # Normal input
         elif len(keyName) == 1:
             if not isReleased:
-                self.state.inputBuffer += keyName
\ No newline at end of file
+                self.state.inputBuffer += keyName
```

Comment:
Please refrain from unnecessary line-noise. Learn to use `git add -p` or your IDE's equivalent.

Reply:
but pep8 :(

Reply:
Pep8 says files should end with a `\n` (like vim does), I'm all for it.

Reducing line noise is about making it easier to review for the reviewer by removing all the noise that has nothing to do with the PR. If you touch a function or create a new file, please follow PEP8 but if you do PEP8 enforcement changes in the same commit / PR as another feature then it makes review harder to do.

When an existing code-base already has lacking PEP8 enforcement, using a tool to convert it all to PEP8 compliant is another case of noise that is added for little benefit. Most of the time when trying to find commit introducing problems with `git log -p` reverse search, you will always stumble on these big "re-indent" commits and it slows down your quest. For such codebases the re-indent / pep8 enforcement should be done gradually as you are modifying the code for another purpose.

Of course, this is a mix of experience and personal preference, your mileage may vary.